examples/pytorch/schnet/mgcn.py

```diff
@@ -0,0 +1,113 @@
+import dgl
+import torch as th
+import torch.nn as nn
+from layers import AtomEmbedding, RBFLayer, EdgeEmbedding, \
+    MultiLevelInteraction
+
+
+class MGCNModel(nn.Module):
+    """
+  MGCN Model from:
+    Chengqiang Lu, et al.
+    Molecular Property Prediction: A Multilevel
+    Quantum Interactions Modeling Perspective. (AAAI'2019)
+  """
+
+    def __init__(self,
+                 dim=128,
+                 output_dim=1,
+                 edge_dim=128,
+                 cutoff=5.0,
+                 width=1,
+                 n_conv=3,
+                 norm=False,
+                 atom_ref=None,
+                 pre_train=None):
+        """
+        Args:
+            dim: dimension of feature maps
+            cutoff: the maximum distance between nodes
+            width: width in the RBF layer
+            atom_ref: atom reference
+                      used as the initial value of atom embeddings,
+                      or set to None with random initialization
+            n_conv: number of convolutional layers
+            norm: normalization
+            pre_train: pre_trained node embeddings
+        """
```

Comment:
If you want to keep the description of the arguments, describe all of them in the order they appear.